lblod/ember-rdfa-editor-lblod-plugins
# Update editor to rdfa-backwards-compatibility build

### Overview
This PR updates the editor to the rdfa-backwards-compatible build (https://github.com/lblod/ember-rdfa-editor/pull/1155)
This build includes some API changes:
- `rdfaAttrSpec` is now a function
- usage of the `...WithConfig` configurable nodespecs

This PR also includes some changes to the citation plugin to support the `rdfaAware` system.

##### connected issues and PRs:
https://github.com/lblod/ember-rdfa-editor/pull/1155

### How to test/reproduce
- `pnpm start`
- Visit the different dummy pages, all node-specs should be in `rdfaAware` mode and the rdfa-features should work as expected

### Challenges/uncertainties
It does not seem very useful to also convert all the nodespecs provided by this package to be backwards compatible.

### Checks PR readiness
